packages/turf-transform-scale/README.md

```diff
@@ -0,0 +1,52 @@
+# @turf/transform-scale
+
+# scale
+
+Moves any geojson Feature or Geometry of a specified distance along a Rhumb Line
```

Comment:
@DenisCarriere probably you can find a better description for the module, explaining the Rhumb "motion" of the pointa; this however is the `translate` description